ui/src/pages/ShoeMap.tsx

```diff
@@ -160,7 +161,13 @@ export const ShoeMap: React.FC = () => {
             </StyledHelpIcon>
           </Control>
           <Control position="bottomleft">
-            <StyledLogo></StyledLogo>
+            <a
+              href="https://theshoeproject.online/"
```

Comment:
Would it make more sense to direct to https://theshoeproject.online/our-stories where the map is originally linked?

Reply:
@jenna-hirji thoughts?

Reply:
confirmed with Anna, let's do https://theshoeproject.online/our-stories